docs/en/stack/ml/df-analytics/evaluatedf-api.asciidoc

```diff
@@ -99,3 +99,38 @@ closer to 1, the better the algorithm performance.
 The {evaluatedf-api} can return the false positive rate (`fpr`) and the true 
 positive rate (`tpr`) at the different threshold levels, so you can visualize 
 the algorithm performance by using these values.
+
+
+[discrete]
+[[ml-dfanalytics-regression-evaluation]]
+=== {regression-cap} evaluation
+
+This evaluation type is suitable for evaluating {regression} models. The 
+{regression} evaluation type offers the following metrics to evaluate the model 
+performance:
+
+* Mean squared error (MSE)
+* R-squared (R^2^)
+
+
+[discrete]
+[[ml-dfanalytics-mse]]
+==== Mean squared error
+
+You can measure how well the {reganalysis} model is performing by computing the 
+average squared sum of the difference between the true value and the value that 
+the {regression} model predicted. (Avg (predicted value-actual value)^2^).
+This value is called mean squared error.
+
+
+[discrete]
+[[ml-dfanalytics-r-sqared]]
+==== R-squared
+
+Another evaluation metrics for {reganalysis} is R-squared (R^2^). It represents 
+the goodness of fit and measures how much of the variation in the data the 
+predictions are able to explain. The value of R^2^ ranges from 0 to 1, where 1 
```

Comment:
The values can be less than 0, i.e. if the model predicts the values further from true values than the mean on average. For example, a predictor of mean + 10 would have a negative R^2.
```suggestion
predictions are able to explain. The value of R^2^ are less than or equal to 1, where 1 
```